libs/features/pages/content-development/src/lib/lib.routes.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
```

Comment:
rename this file to `content-development.routing.ts`

Reply:
Changes requested have been made